modules/campaignion_match_to_target_type/src/ChannelEmailNoSend.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Drupal\campaignion_match_to_target_type;
+
+use Drupal\campaignion_email_to_target\Channel\Email;
+use Drupal\little_helpers\Webform\Submission;
+
+/**
+ * Channel that allows to compose an email but don’t send it.
+ */
+class ChannelEmailNoSend extends Email {
+
+  /**
+   * Don’t do anything when the form is submitted.
+   */
+  public function send() {
+    return TRUE;
+  }
+
+  /**
+   * Don’t filter targets and messages.
+   */
+  public function filterPairs(array $pairs, Submission $submission, bool $test_mode) {
+    return $pairs;
+  }
+
+}
```

Comment:
This class should better be put into the [campaignion](https://github.com/moreonion/campaignion) project. Perhaps in the email to target module. Then we don’t need to copy&paste it into all profiles.